include/libmw/defs.h

```diff
@@ -165,7 +165,11 @@ struct Coin
 
```

Comment:
Remove MWEBAddress and PartialTransaction?

Reply:
Actually, MWEBAddress is still used. I've removed PartialTransaction.